.github/workflows/tests_cmake_testsuite.yml

```diff
@@ -148,3 +148,34 @@ jobs:
     needs: [preparation, sc_p4est_tests, t8code_tests, t8code_linkage_tests, t8code_api_tests, t8code_w_shipped_submodules_tests]
     with:
       LESS_TESTS: ${{ github.event_name == 'pull_request'  }}
+  
+  check_commit_message:
+    runs-on: ubuntu-latest
+    outputs: 
+      trigger_version_update: ${{ steps.check_commit_message.outputs.VERSION_UPDATE }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: ${{ github.event.pull_request.head.repo.full_name }}
+        ref: ${{ github.event.pull_request.head.ref }}
+        fetch-depth: 0
+    - name: Check commit message
+      id: check_commit_message
+      run: |
+        COMMIT_MESSAGE=$(git log -1 --pretty=%B)
+        echo "message: $COMMIT_MESSAGE"
+        if [[ "$COMMIT_MESSAGE" == "Update Version" ]]; then
+          echo "Update version"
+          echo "VERSION_UPDATE=true" >> $GITHUB_OUTPUT
```

Comment:
Can we also check if the commit is empty?

Reply:
yes